File: Array_and_Matrix/Easy/1672.Richest_Customer_Wealth/solution.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        wealth = 0
        for row in range(len(accounts)):
            sum = 0
            for column in range(len(accounts[row])):
                sum += accounts[row][column]
            if sum > wealth:
                wealth = sum
        return (wealth)

File: Array_and_Matrix/Easy/1672.Richest_Customer_Wealth/solution.c
int maximumWealth(int** accounts, int accountsSize, int* accountsColSize) 
{
    int row, column, wealth = 0, sum;

    for (row = 0; row < accountsSize; row++)
    {
        sum = 0;
        for (column = 0; column < accountsColSize[row]; column++)
        {
            sum += accounts[row][column];
        }
        if (sum > wealth)
        {
            wealth = sum;
        }
    }
    return (wealth);
}